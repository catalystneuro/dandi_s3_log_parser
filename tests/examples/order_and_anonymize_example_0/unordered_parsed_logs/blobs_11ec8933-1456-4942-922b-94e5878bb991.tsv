timestamp	bytes_sent	ip_address
2022-07-01 05:06:35	512	192.0.2.0
2021-05-21 05:06:35	1234	190.154.159.92
2022-11-04 05:06:35	141424	82.195.200.188
2020-02-24 05:06:35	124	187.176.214.149
