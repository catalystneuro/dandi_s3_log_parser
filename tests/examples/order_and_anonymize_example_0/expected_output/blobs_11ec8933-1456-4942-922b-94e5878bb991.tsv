	timestamp	bytes_sent	region
0	2020-02-24 05:06:35	124	MX/Nuevo Le√≥n
1	2021-05-21 05:06:35	1234	EC/Santa Elena
2	2022-07-01 05:06:35	512	unknown
3	2022-11-04 05:06:35	141424	FI/Uusimaa
